usr/share/rear/rescue/DP/default/450_prepare_omni_xinetd.sh

```diff
@@ -1,8 +1,15 @@
+# 450_prepare_omni_systemd.sh
+# Make sure Data Protector INET gets included in the rescue image (if xinetd is used)
+
+if [ -r "/etc/xinetd.d/omni" ]; then
```

Comment:
Same as https://github.com/rear/rear/pull/2443/files#r447483241
so instead of
```
if [ -r "/etc/xinetd.d/omni" ]; then
...
fi
```
you may use
```
# Nothing to do when xinetd is not used:
test -r "/etc/xinetd.d/omni" || return 0
```